packages/sirius-web/backend/sirius-web-application/src/main/java/org/eclipse/sirius/web/application/document/controllers/MutationUploadDocumentDataFetcher.java

```diff
@@ -69,12 +71,17 @@ public CompletableFuture<IPayload> get(DataFetchingEnvironment environment) thro
                 .filter(UploadFile.class::isInstance)
                 .map(UploadFile.class::cast);
 
-        if (optionalId.isPresent() && optionalEditingContextId.isPresent() && optionalFile.isPresent()) {
+        var optionalReadOnly = Optional.ofNullable(inputArgument.get(READ_ONLY))
+                .filter(Boolean.class::isInstance)
+                .map(Boolean.class::cast);
+
+        if (optionalId.isPresent() && optionalEditingContextId.isPresent() && optionalFile.isPresent() && optionalReadOnly.isPresent()) {
```

Comment:
The schema indicates the new attribute is optional (`readOnly: Boolean`), but here the mutation request is ignored if it os not explicitly set.

Reply:
I made the parameter mandatory. I don't see a good reason to make it optional, since all the places where this API is used has been updated to provide an explicit value.
I updated the _breaking changes_ section of the changelog to reflect this.

Let me know if you think this should be optional, I'll revert my change and implement the fix you provide below.

Reply:
> Let me know if you think this should be optional

I tend to hate optional stuff with a passion 🔥